subscripts/script_1.m

```diff
@@ -1,10 +1,10 @@
 %% script_1
 
-%read in data
-data = csvread('data/peach.csv');
-%plot data
-p = patch(data(:,1), data(:,2), 'orange');
-p.LineWidth = 2;
+% %read in data
+% data = csvread('data/peach.csv');
+% %plot data
+% p = patch(data(:,1), data(:,2), 'orange');
+% p.LineWidth = 2;
```

Comment:
I know you need your solution uncommented to check that it runs, but let's make sure that what we give to the attendees has the broken code uncommented and both solutions commented. So, for all these subscripts, comment out the solutions and uncomment the broken code.

Reply:
fixed w/ be6d557